.kokoro/dependencies.sh

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -eo pipefail
+set -o pipefail
```

Comment:
this would allow `mvn -B dependency:analyze -DfailOnWarning=true` to not fail the build if it finds issues

Reply:
ok -- I put it back